### PR TITLE
Asset stac docs

### PR DESCRIPTION
**Pull request by chrieke**
Additional documentation for new `storage` STAC asset functionality.

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [x] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

**Review by andher1802 — APPROVED** [review]
LGTM

File: docs/storage.md [hunk 1]
```diff
@@ -14,25 +14,42 @@ up42.authenticate(
 )
 ```
 
+## **Access Orders**
 
-## **Access & Download Assets**
-
-Get a list of all order or asset objects in your user storage.
+Get a list of all orders in your user storage. Then access or [download](#download-assets)) the image assets of an
```
Comment (seedlit) [inline docs/storage.md:19]:
```suggestion
Get a list of all orders in your user storage. Then access or [download](#download-assets) the image assets of an 
```

File: docs/storage.md [hunk 2]
```diff
@@ -45,4 +62,24 @@ for asset in assets:
...
+up42_pystac_client = storage.pystac_client
+```
+
+With the `up42_pystac_client`, it is possible to run the regular pystac client operations (e.g. to go through the UP42 assets AKA UP42 pystac-collections in your account). 
+
+```python
+up42_stac_collections = up42_pystac_client.get_collections()
+```
+
+Check the documentation for reference about UP42 stac definitions.
```
Comment (seedlit) [inline docs/storage.md:80]: maybe consider adding a hyperlink here pointing to the `UP42 STAC definitions`.

**Review by seedlit — APPROVED** [review]
Nice!
LGTM!

File: docs/storage.md (hunk 2 above)
Comment (PelJon) [reply docs/storage.md:80]: Where do we show the actual PySTAC Client capabilities - did I miss that somehow?

**Pull request merged**